### DISCUSSION TITLE
Buggy comment image downscaling

### DISCUSSION THREAD

**Discussion by ghost** [community/community#4187] [Mobile]
Images with a set width do not resize appropriately on mobile web. For example, the screenshot attached to https://github.com/microsoft/vscode-extension-samples/issues/448 on mobile web:

![Screenshot_20210615-150644.jpg](https://user-images.githubusercontent.com/82655227/122067883-d6d19480-cdeb-11eb-8733-edd7f188a57e.jpg)

**Comment by amrehman** [comment]
@4086606 
Thanks a lot for getting in touch.
Can you please confirm for me if you are using the mobile website on your phone's browser or are you using one of our native Android/iOS Apps.

Looking forward to hear back from you!

**Reply by ghost** [reply]
The bug appears on both platforms unfortunately, native Android and mobile web browser (Chrome)

**Reply by amrehman** [reply]
thanks a lot for getting back so quickly.
I will create a bug for our team that works on the web app.

In the mean time, can I tempt you into trying out our native mobile Apps 

You can find it on the Apple App Store [here](https://apps.apple.com/us/app/github/id1477376905)

And the Android Play store [here](https://play.google.com/store/apps/details?id=com.github.android&hl=en&gl=US)

It seems the issue also repros on our native iOS App :( but feel free to explore the app and provide us feedback :) 
<img width="310" alt="image" src="https://user-images.githubusercontent.com/17592342/123049301-b1eaac00-d3ff-11eb-894f-2cc9f2d475fc.png">

**Reply by ghost** [reply]
Yeah your app is great! I've got the Android version and I've found all the new features super useful
